### PR TITLE
New option in files_external:list --mount-options

### PR DESCRIPTION
**Pull request by mmattel**
References core https://github.com/owncloud/core/pull/36420 (merged)

Update of the `files_external:list` command, added a new option: `--mount-options` plus example.
Backport to 10.3 is needed when the new Release of 10.3.x is published

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 1]
```diff
@@ -917,6 +918,25 @@ be added as system mount.
 +----------+------------------+----------+
 ----
 
+===== Example 2
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} files_external:list  --mount-options --output=json_pretty
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: What about:

```
{occ-command-example-prefix} files_external:list \
    --mount-options \
    --output=json_pretty
```

Perhaps it renders nicely enough as is, however.

**Comment by mmattel** [comment]
Incorporated your suggestion. Looks better :smile:

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 2]
```diff
@@ -882,7 +882,15 @@ List configured mounts.
 
 [source,console]
 ----
-files_external:list [--show-password] [--full] [-a|--all] [-s|--short] [--] [<user_id>]
+files_external:list
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: Sorry for any confusion caused, but I only meant to have linebreaks for the code example, not for anything else.

Comment (mmattel) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: We have this look and feel also on other locations where the list or length of options and arguments is long and therefore broke it into multi lines. Example see https://doc.owncloud.org/server/10.3/admin_manual/configuration/server/occ_command.html#files_externalcreate

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: Fair enough.

**Pull request merged**